plugins/blip/basics/logic/state.js

```diff
@@ -148,8 +154,19 @@ var basicsState = {
       index: 3,
       noDataMessage: 'Infusion site changes are not yet available for all pumps. Coming soon!',
       togglable: togglableState.off,
+      selector: SiteChangeSelector,
+      selectorOptions: {
+        primary: { key: constants.SITE_CHANGE_RESERVOIR, label: 'Reservoir Change' },
```

Comment:
Probably! Though it's currently not displayed anywhere, and would be [variable-ized based on pump](https://github.com/tidepool-org/tideline/pull/299/files#diff-45ff9e6756044f644ca92e6381755b8fR164), if it were to be shown.